index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Real </title>
+</head>
+
+<body>
+    <header id="header"> 
+        <h1>Kano-Bruda Real Estate</h1>
+        <img src="./vector-illustration-cool-detailed-red-260nw-94498447.jpg" id="header-img" alt="">
+    <nav id="nav-bar">
+        <ul>
+            <a class="nav-link" href="#Overview"><li>Overview</li></a>
+                    <a class="nav-link" href="#view_Homes"><li>Locations</li></a>
+                    <a class="nav-link" href="#homws"><li>Customers</li></a>
+                    
+        </ul>
+    </nav>
+    </header>
+    
+    <form  id="form" action="">
+        email: <input id="email" type="text">
+<input type="submit" id="submit" >
+    </form>
```

Comment:
It is a good practice to include a method attribute on your form. This is done to specify how to send your form data to the server. Please insure you include a valid URL inside the form action attribute, so that you define where your data will go after form submission.

Reply:
Nice job being polite, giving reasons for the requested change, and phrasing your comments as suggestions